src/Umbraco.Core/Configuration/GlobalSettings.cs

```diff
@@ -70,7 +70,8 @@ public static bool HasSmtpServerConfigured(string appPath)
 
             var config = WebConfigurationManager.OpenWebConfiguration(appPath);
             var settings = (MailSettingsSectionGroup)config.GetSectionGroup("system.net/mailSettings");
-            if (settings == null || settings.Smtp == null) return false;
+            // note: "noreply@example.com" is/was the sample SMTP from email - we'll regard that as "not configured"
+            if (settings == null || settings.Smtp == null || "noreply@example.com".Equals(settings.Smtp.From, StringComparison.OrdinalIgnoreCase)) return false;

```

Comment:
Personally I would have done it like this.

`settigns.Smtp.From.Equals("noreply@example.com", StringComparison.OrdinalIgnoreCase)`

Somehow my mind blows up seeing it the other way around...but that is a personal preference.

I have tested it and can't find anything that breaks.

Reply:
@dawoe normally I'd do it the other way around too... but this way is actually null reference proof, thus no need to null check `settings.Smtp.From` first.